config/locales/en.yml

```diff
@@ -64,6 +64,8 @@ en:
 
   resources:
     caption: "The following resources were created as part of your deployment:"
+    cleanup:
+      caption: "If you don't need this cluster anymore, we can help you clean up. We can generate a list of commands you can run later to clean up, or it can be done automatically from this instance."
```

Comment:
we could be a bit more explicit 

we could help you clean all the resources up

so user does not think it will clean only the cluster ? (mentioned just before)

Reply:
[75ee814](https://github.com/SUSE-Enceladus/suse-rancher-setup/pull/195/commits/75ee8147b85c0a4b17c0c95a163f5a6b5d4529c4)